pkg/api/ldap.go

```diff
@@ -216,8 +216,11 @@ func (lc *LDAPClient) Authenticate(username, password string) (bool, map[string]
 	}
 
 	userDN := search.Entries[0].DN
-	userAttributes := search.Entries[0].Attributes[0]
-	userGroups := userAttributes.Values
+	var userGroups []string
+	if len(search.Entries[0].Attributes) > 0 {
```

Comment:
My understanding is this a configuration issue which would happen in all cases of authentication if the zot or ldap server configuration is not changed by the zot admin or on the other side the ldap admin.

There's no way right now for the zot to know why permissions are not working because of this mismatched configuration. We should not let this pass silently.

I think there should be a warning in the logs in case the user group attribute set in the zot config does not match the actual ldap configuration.